.github/workflows/v2-tests-e2e.yml

```
@@ -65,7 +65,7 @@ jobs:
         run: |
           cd test-website
           yarn set version berry
```

```suggestion
          yarn set version berry
          yarn set version canary
```
As @SamChou19815 mentioned the issue has been fixed in a canary release

---

Thanks, will try that! 
Is this normal however that I also have this error in Yarn 2.4.2? Both v2/v3 are affected?

Not sure it's even related but like a month ago, all our Yarn 2 e2e tests on GH actions would timeout on the install step, no idea why: https://github.com/facebook/docusaurus/pull/5342/checks?check_run_id=3304019232
It looks like generating our example app inside the monorepo, despite not being a monorepo workspace package, makes this timeout. If I move the package outside of the monorepo (which imho we should do, it's a more realistic setup of what the users will end-up with) then those timeouts disappear. The CI logs aren't really helpful unfortunately.

---

Yes, there was a bug in our fs patch (https://github.com/yarnpkg/berry/issues/3234) and something in the dependency tree started running into it.

Probably unrelated but interesting, I would expect it to at least log something. Have you been able to reproduce it locally?

---

The timeout can't be reproduced locally. The GH action step freeze (without logs) for 30min before our timeout is reached, and only then some logs become available but do not show anything useful (https://github.com/facebook/docusaurus/pull/5342/checks?check_run_id=3304019232). Maybe it's a GH action bug.